### PR TITLE
Ruleset change tests (resolves #5844)

### LINKED ISSUE #5844: Only do fetch test for ruleset changing PRs
As mentioned in https://github.com/EFForg/https-everywhere/pull/5626#issuecomment-235990461, it would make sense to change our Travis setup so that only the fetch tests are carried out for a given PR containing only ruleset changes, the main benefit being a dramatic decrease in Travis test durations.

### PR DESCRIPTION
**Pull request by Hainish**
… checking (resolves #5844)

### CODE REVIEW

**Comment by Hainish** [comment]
Ready for review @cooperq @jsha

File: test/travis-ruleset-fetch.sh [hunk 1]
```diff
@@ -22,11 +22,25 @@ git remote add upstream-for-travis https://github.com/EFForg/https-everywhere.gi
...
+  echo >&2 "Running main test suite."
+  ./test/firefox.sh
+  ./test/chromium.sh
+fi
 # Only run test if something has changed.
 if [ "$RULESETS_CHANGED" ]; then
-  echo >&2 "Ruleset database has changed. Testing test URLs in all changed rulesets."
+  echo >&2 "Ruleset database has changed."
+
+  echo >&2 "Performing comprehensive coverge test."
```
Comment (fuglede) [inline test/travis-ruleset-fetch.sh]: Here's a typo.

Comment (Hainish) [inline test/travis-ruleset-fetch.sh]: Thanks for catching it!

**Comment by cooperq** [comment]
lgtm

**Pull request merged**